addons/mass_mailing/wizard/mailing_list_merge.py

```diff
@@ -22,7 +22,7 @@ def default_get(self, fields):
         res = super(MassMailingListMerge, self).default_get(fields)
         src_list_ids = self.env.context.get('active_ids')
```

Comment:
While we are at it, we could check that active_model is effectively mailing.list .